test/cpp_extensions/cudnn_extension.cpp

```diff
@@ -0,0 +1,73 @@
+#include <torch/torch.h>
+#include <cuda.h>
+#include <torch/torch.h>
+#include <ATen/ATen.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/cudnn/Descriptors.h>
+#include <ATen/cudnn/Exceptions.h>
+#include <ATen/cudnn/Handles.h>
+#include <torch/csrc/cuda/cuda_check.h>
+#include <torch/torch.h>
+#include <iostream>
+
+using namespace at;
+using namespace std;
+using namespace at::native;
+const char *cudnn_relu_name = "cudnn_relu";
+
+// Check arguments to cudnn_relu
+int cudnn_relu_check(const Tensor &inputs, const Tensor &outputs) {
+  // Create TensorArgs
+  TensorArg arg_inputs(inputs, "inputs", 0);
+  TensorArg arg_outputs(outputs, "outputs", 1);
+  // Check arguments
+  checkContiguous(cudnn_relu_name, arg_inputs);
+  checkScalarType(cudnn_relu_name, arg_inputs, ScalarType::Float);                      \
+  checkBackend(cudnn_relu_name, arg_inputs.tensor, Backend::CUDA);
+  checkContiguous(cudnn_relu_name, arg_outputs);
+  checkScalarType(cudnn_relu_name, arg_outputs, ScalarType::Float);                      \
+  checkBackend(cudnn_relu_name, arg_outputs.tensor, Backend::CUDA);
+  checkSameSize(cudnn_relu_name, arg_inputs, arg_outputs);
+  return 0;
+}
+
+int cudnn_relu(const Tensor& inputs, const Tensor& outputs) {
+  //Check inputs
```

Comment:
Could you clang format this file? Again, just trying to have consistent coding style in the C++ extensions. There is a `.clang-format` file in the root of this repo. You need only run `clang-format -i <this-file>` and it'll auto-format it for you.

Reply:
sure. That sounds like something that CI should check as well. Do you want to start an issue to work that into CI?

Reply:
Yes that is something I've planned for July :)